acstools/findsat_mrt.py

```diff
@@ -0,0 +1,1129 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+This module contains a class called trailfinder that is used to identify 
+satellite trails and/or other linear features in astronomical image data. To 
+accomplish this goal, the Median Radon Transform (MRT) is calculated for an image. 
+Point sources are then extracted from the MRT and filtered to yield a final 
+catalog of trails. These trails can then be used to create a mask.
+
+Example 1: Identificaation of trails in an ACS/WFC image, j97006j5q_flc.fits (4th extension)
```

Comment:
Regarding the interactive plotting, would it be ok to have python check if interactive mode is running, e.g.:
`import matplotlib as mpl
mpl.is_interactive()`
If that returns false, the plots can be written out to file, otherwise, they could appear as popups (I could also add an option to both plot on screen and save to disk)

Reply:
This solution sounds good to me.